public/translations/de.json

```diff
@@ -44,6 +44,7 @@
   "tonies": {
     "navigationTitle": "Tonies",
     "title": "Tonies",
+    "unsetTonie": "Undefinierter Tonie",
```

Comment:
Sounds kind of negative. Would only state „Tonie“. Same in English

Reply:
I think it would be enough just naming it Tonie or Tonie/Tag.